Intersect.Editor/Forms/frmMain.cs

```diff
@@ -367,11 +367,11 @@ private void FrmMain_KeyDown(object sender, KeyEventArgs e)
                         toolStripBtnFlipHorizontal_Click(null, null);
                         break;
 
-                    case Keys.F: // Fill Layer Tool.
+                    case Keys.F: // Layer Fill Tool.
```

Comment:
Since it's just a comment I don't really care but I would personally revert this